motoko/life/package.json

```diff
@@ -1,31 +1,50 @@
 {
   "name": "life_assets",
   "version": "0.1.0",
-  "description": "",
-  "keywords": [],
+  "description": "Internet Computer starter application",
+  "keywords": [
+    "Internet Computer",
+    "Motoko",
+    "JavaScript",
+    "Canister"
+  ],
   "scripts": {
-    "build": "webpack"
+    "build": "webpack",
+    "prebuild": "npm run copy:types",
+    "start": "webpack serve --mode development --env development",
+    "prestart": "npm run copy:types",
+    "copy:types": "rsync -avr .dfx/$(echo ${DFX_NETWORK:-'**'})/canisters/life/** --exclude='assets/' --exclude='idl/' --exclude='*.wasm' --delete src/declarations"
```

Comment:
why is all this necessary now - is that just because of the weird setup of the life demo to showcase upgrade?

Reply:
This is to make the examples more directly comparable to the `dfx new` project

Reply:
Ok